org.dupot.nix-disk-manager.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.dupot.nix-disk-manager
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: nix_disk_manager
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
`--talk-name=org.freedesktop.Flatpak` why is this needed?

Reply:
i made a mistake i removed it